### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by zelfroster**
Fixed
- for more than 1 recipient, recipient name was getting added on next line
- input field password autofocus and Login when pressing Enter on keyboard on login page
- When mail contains no Author/from, the date will be added to the wrong field

### CODE REVIEW

**Comment by zelfroster** [comment]
Please don't merge this PR untill some time as I will be adding more fixes in this one.

**Comment by defnax** [comment]
big importend on mails is reply button, was nice this is working on release

File: webui-src/assets/index.html [hunk 1]
```diff
@@ -9,19 +10,22 @@
...
-        }
-    </script>
-    <noscript>Sorry but JavaScript is needed here! Please enable it in your browser.</noscript>
+        <h3>
+            Javascript not found!
+        </h3>
+        <script type="text/javascript">
+            function load_ui() {
+                var main = require('main');
+                document.getElementById('password').focus();
```
Comment (rottencandy) [inline webui-src/assets/index.html:22]: Why not use the autofocus attribute

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

Comment (zelfroster) [reply webui-src/assets/index.html:22]: For some reason, autofocus is not working.
So, I tried to do it with JavaScript 😅

Comment (rottencandy) [reply webui-src/assets/index.html:22]: Ok it's better to do this in the oncreate lifecycle method

Comment (zelfroster) [reply webui-src/assets/index.html]: Fixed it, Thanks @rottencandy

**Pull request merged**